_overviews/cheatsheets/index.md

```diff
@@ -594,25 +590,25 @@ println(upper.getOrElse(""))</code></pre></td>
   case Some(x) if !f(x) =&gt; Some(x)
   case _                =&gt; None
 }</code></pre></td>
-      <td>Optional value doesn't satisfy predicate</td>
+      <td>Optional value doesn't satisfy predicate.</td>
     </tr>
     <tr>
       <td><pre class="highlight"><code>option.exists(f(_))</code></pre>
       <em><strong>same as</strong></em>
       <pre class="highlight"><code>option match {
-  case Some(x) if f(x) =&gt; true
-  case _               =&gt; false
+  case Some(x) =&gt; f(x)
+  case None    =&gt; false
```

Comment:
This is also a true implementation of `exists`.  There's no reason to have the boolean guard condition.  However, I think the longer version helps with readability.